src/main/scala/fi/oph/koski/migri/MigriServlet.scala

```diff
@@ -14,6 +14,24 @@ class MigriServlet(implicit val application: KoskiApplication) extends KoskiSpec
   lazy val migriService =
     if (application.config.getString("opintopolku.virkailija.url") == "mock") new MockMigriService else new RemoteMigriService
 
+  // Overridetaan RequiresLuovutuspalvelu-traitin funktio.
```

Comment:
Minun käsitykseni oli/on, että vain tuo valinta-API pitäisi disabloida: nämä muut vuosi sitten kesällä tms. tehdyt API:t saavat toimia, ja saattavat olla käytössäkin.

Reply:
Tarkistin Kaisalta, kummatkin ois esteettävä.
"ja olikos tää nyt siis niin, että migrillä on soppari voimassa ton vanhan API:n tuottamaan dataan, mutta ei tohon uudempaan, eli kaikki noi uuden API:n polut pitäis tuotannossa estää?

Ja just näin eli vanhaan rajapintaan on soppari voimassa, uuteen ei ole vielä tehty ja tulee siksi estää niin valintatietojen kuin luovutuspalvelun osalta tuotannossa"